src/patternfly/components/Content/styles.scss

```diff
@@ -2,17 +2,17 @@
 
 .pf-c-content {
   a {
-    color: $pf-global--link--Color;
-    text-decoration: $pf-global--link--TextDecoration;
+    color: $global--link--Color;
+    text-decoration: $global--link--TextDecoration;
```

Comment:
Are these suppose to be custom properties or sass variables in this PR? or does it matter?